tests/target/configs/struct_field_align_threshold/20.rs

```diff
@@ -41,9 +41,9 @@ pub struct Foo {
     f: SomeType, // Comment beside a field
     // Comment on a field
     #[AnAttribute]
-    g: SomeOtherType,
+    g:     SomeOtherType,
```

Comment:
This seems suspicious, I think we should not align items that are separated by comment.

Reply:
from https://github.com/rust-lang/rustfmt/pull/3513#issuecomment-500213375 groups are now defined by blank lines.